bugbug/models/bug.py

```diff
@@ -50,12 +51,14 @@ def __init__(self, lemmatization=False):
 
                 ('title', self.text_vectorizer(), 'title'),
 
-                ('first_comment', self.text_vectorizer(), 'first_comment'),
+                ('first_comment', self.text_vectorizer(min_df=0.001), 'first_comment'),
+
+                ('comments', self.text_vectorizer(min_df=0.001), 'comments'),
```

Comment:
Let's remove the `min_df` changes (both here and in tracking.py), I'll set them myself by checking what we can use on my machine and on the AWS instance we are using.